src/publisher.ts

```diff
@@ -34,6 +36,16 @@ export class Publisher {
 
 		if (options.pactFilesOrDirs) {
 			checkTypes.assert.array.of.string(options.pactFilesOrDirs);
+			for(let i = 0, len = options.pactFilesOrDirs.length; i<len; i++) {
```

Comment:
Any reason this couldn't be something like `options.pactFilesOrDirs.forEach((item) => { ... } )` ?